contrib/babelfishpg_tsql/src/pltsql_bulkcopy.c

```diff
@@ -683,6 +686,12 @@ ExecuteBulkCopy(BulkCopyState cstate, int rowCount, int colCount,
 					{
 						myslot->tts_values[i] = Int64GetDatum(nextval_internal(cstate->seqid, true));
 					}
+					else if (cstate->rv_index == i)
+					{
+						Expr *defexpr = (Expr *) build_column_default(cstate->rel, i + 1);
+						defexpr = expression_planner(defexpr);
+						myslot->tts_values[i] = ExecEvalExpr(ExecInitExpr(defexpr, NULL), econtext, &myslot->tts_isnull[i]);
+					}
```

Comment:
why do we need this?
Row version will act like a column with default values column. So this will be taken care of on its own [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/776/files?diff=unified&w=0#diff-6abbce3443c2d108360f1a0e87bab73624ac7353302c523840d7235e8bdfd354R734)